webview-ui/src/components/chat/auto-approve-menu/AutoApproveModal.tsx

```diff
@@ -84,6 +96,13 @@ const AutoApproveModal: React.FC<AutoApproveModalProps> = ({ isVisible, setIsVis
 						style={{ textDecoration: "underline" }}>
 						Configure notification settings
 					</span>
+					{" | "}
+					<span
+						className="underline cursor-pointer hover:text-foreground"
+						onClick={handleDocsLinkClick}
+						style={{ textDecoration: "underline" }}>
+						Docs
+					</span>
```

Comment:
we can just use the link element without an onclick handler?
```suggestion
					<a className="text-link hover:text-link-hover" href="https://docs.cline.bot/features/auto-approve#auto-approve">
						Docs
					</a>
```